test/framework/toy_build.py

```diff
@@ -2855,9 +2855,8 @@ def test_fix_shebang(self):
                             "Pattern '%s' found in %s: %s" % (bash_shebang_regex.pattern, bashbin_path, bashbin_txt))
 
         # now test with a custom env command
-        init_config(build_options={'env_for_shebang': "/usr/bin/env -S"})
-
-        self.test_toy_build(ec_file=test_ec, raise_error=True)
+        extra_args = ['--env-for-shebang=/usr/bin/env -S']
```

Comment:
probably should quote that command

Reply:
apparently not

Reply:
Nope, then the test fails like this, because the `"` are included with the value:
```
FAIL: test_fix_shebang (__main__.ToyBuildTest)
Test use of fix_python_shebang_for & co.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Volumes/work/easybuild-framework/test/framework/toy_build.py", line 2880, in test_fix_shebang
    "Pattern '%s' found in %s: %s" % (py_shebang_regex.pattern, pybin_path, pybin_txt))
AssertionError: None is not true : Pattern '^#!/usr/bin/env -S python\n# test$' found in /var/folders/zn/3p3vftw90hs58m1gn61sc7b80000gn/T/eb-noryBe/eb-qvCMJr/tmpY2HzEB/software/toy/0.0/bin/t1.py: #!"/usr/bin/env -S" python
# test
```